src/xtgeo/xyz/polygons.py

```diff
@@ -614,20 +638,32 @@ def get_shapely_objects(self):
 
         return spolys
 
+    @inherit_docstring(inherit_from=XYZ.get_boundary)
     def get_boundary(self):
-        """Get the XYZ window (boundaries) of the instance.
+        return super().get_boundary()
+
+    def simplify(
+        self, tolerance: Optional[float] = 0.1, preserve_topology: Optional[bool] = True
```

Comment:
Never really got the preserve_topology option to do anything.. but probably due to my dataset 😄 but this function gave nice results 👍